MinionMovies/DetailsView/DetailsViewDataProvider.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DetailsViewDataProvider.swift
+//  MinionMovies
+//
+//  Created by Vitor Costa on 14/05/20.
+//  Copyright © 2020 Vitor Costa. All rights reserved.
+//
+
+import Foundation
+
+protocol DetailsViewDataProviderProtocol {
+    func add(item: FavMovieDB)
+    func delete(item: String)
+    func check(item: String) -> Bool
```

Comment:
delete(item: String)... o que é item do tipo string? pra mim que to lendo não sei o que eu to deletando com um string chamado item